hamocc/mo_ocprod.F90

```diff
@@ -490,10 +501,17 @@ subroutine ocprod(kpie,kpje,kpke,kbnd,pdlxp,pdlyp,pddpo,omask,ptho,pi_ph,psao,pp
             endif
             ocetra(i,j,k,idet) = ocetra(i,j,k,idet)+export
             ocetra(i,j,k,idms) = ocetra(i,j,k,idms)+dmsprod-dms_bac-dms_uv
-            ocetra(i,j,k,isco212) = ocetra(i,j,k,isco212)-delcar+rcar*dtr
+            ocetra(i,j,k,isco212) = ocetra(i,j,k,isco212)-delcar+rcar*dtr ! formerly
+            if (use_r2o) then
+              ocetra(i,j,k,isco212) = ocetra(i,j,k,isco212)+tdoclc_deg*rcar_tdoclc+tdochc_deg*rcar_tdochc
```

Comment:
I am missing the alkalinity change due to DOC remin here. If you need formulas for the change due to different stoichiometry, you can get back to me.

Reply:
+ iron release, similar below